chibuezeayogu/Events-Manager
# ch(postgres): Configure postgres database

**What does the PR do?**
- This PR configures app to use `Postgres` against `sqlite3`

**Description of Task to be completed?**
- Ensures postgres is properly configured

**How should this be manually tested?**
- Clone the repository [Events-Manager](https://github.com/chibuezeayogu/Events-Manager.git) and
- Checkout to  this branch `configure-postres-db-158749852`
- run `rails db:migrate` then `rails db:setup` on the terminal
- ensure `rails db:migrate` and `rails db:setup` are successfully executed 

**What are the relevant pivotal tracker stories?**
- [#158749852](https://www.pivotaltracker.com/story/show/158749852)

**Screenshots**
- None
